test/acceptance/test/add_tests.go

```diff
@@ -360,53 +339,49 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 		})
 
 		By("And I should not see any workload deployed to the cluster", func() {
-			verifyWegoAddCommandWithDryRun(appRepoName, WEGO_DEFAULT_NAMESPACE)
+			verifyWegoAddCommandWithDryRun(tip.appRepoName, WEGO_DEFAULT_NAMESPACE)
 		})
 	})
 
 	// Eventually this test run will include all the remaining un-automated `wego app add` flags.
 	It("Verify 'wego app add' works with user-specified branch, namespace, url", func() {
 		var repoAbsolutePath string
 		private := true
-		appRepoName := "wego-test-app-" + RandString(8)
+		tip := generateTestInputs()
 		branchName := "test-branch-02"
-		uniqueSuffix := RandString(6)
-		appManifestFilePath := getUniqueWorkload("xxyyzz", uniqueSuffix)
-		workloadName := "nginx-" + uniqueSuffix
-		workloadNamespace := "my-nginx-" + uniqueSuffix
-		defaultSshKeyPath := os.Getenv("HOME") + "/.ssh/id_rsa"
+
 		wegoNamespace := "my-space"
-		url := "ssh://git@github.com/" + os.Getenv("GITHUB_ORG") + "/" + appRepoName + ".git"
+		url := "ssh://git@github.com/" + os.Getenv("GITHUB_ORG") + "/" + tip.appRepoName + ".git"
```

Comment:
I like to use `fmt.SprintF()` for long string concatenations: https://golang.org/pkg/fmt/#Sprintf

Purely stylistic, but it make things easier to read IMO.

Reply:
Yeah, I am not happy with it either, let me do it next pass.